model/shop.php

```diff
@@ -331,7 +331,7 @@ function vstupneHtml() {
       'jsSlider'  =>  URL_WEBU.'/soubory/blackarrow/shop/shop-vstupne.js',
       'stav'      =>  $this->u->gcPrihlasen() ?
         $this->vstupne['sum_cena_nakupni'] + $this->vstupnePozde['sum_cena_nakupni'] :
-        60, // výchozí hodnota
+        0, // výchozí hodnota
```

Comment:
Hodil bych to taky do konstanty á la option, něco jako `VYCHOZI_DOBROVOLNE_VSTUPNE` (jestli to teda chápu dobře a je to výchozí částka vstupného - ze šablony jsem toho moc nevykoukal `model/shop-vstupne.xtpl`).